Readme.md

```
@@ -18,6 +18,18 @@ query [Circles](https://www.aboutcircles.com/) protocol events.
     * [circles_query](#circles_query)
     * [circles_events](#circles_events)
     * [eth_subscribe("circles")](#eth_subscribecircles)
+    * [circlesV2_findPath](#circlesv2_findpath)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)